dev/components/videoTile/styles.scss

```diff
@@ -10,6 +10,7 @@
   .photo {
     height: 143px;
     width: 100%;
+    background-image: url('/files/images/photo-video.png')
```

Comment:
Please use more clearest placeholder for video tile (e.g. https://uneplive.unep.org/media/imgs/no_img_video.jpg)